projects/packages/connection/src/class-error-handler.php

```diff
@@ -277,7 +277,8 @@ public function store_error( \WP_Error $error ) {
 		$stored_errors[ $error_code ][ $user_id ] = $error_array;
 
 		// Let's store a maximum of 5 different user ids for each error code.
-		if ( count( $stored_errors[ $error_code ] ) > 5 ) {
+		$error_code_count = is_countable( $stored_errors[ $error_code ] ) ? count( $stored_errors[ $error_code ] ) : 0;
+		if ( count( $error_code_count ) > 5 ) {
```

Comment:
Wouldn't it be `if( $error_code_count > 5 ) { `

Reply:
Yes, it would. Good catch! 🤦